inst/WORDLIST

```diff
@@ -1,11 +1,14 @@
 ald
+decarbonisation
 Decarbonisation
 DegreesInvesting
 dfrac
 dii
+DII
```

Comment:
Are we supposed to use DII or Dii (uppercase or lowercase i)?

Reply:
technically uppercase, 2DII, but people jump around a lot